BatchPreAnalysis.py

```diff
@@ -0,0 +1,139 @@
+"""
```

Comment:
I am wondering if this file was uploaded accidentally, as BatchPreAnalysis.py does not directly interact with the time resolved analysis. You possibly checked out a branch not from the remote "automated-robustness", but from a local more current version which contained changes from other branches. Please make sure to only upload files into the PR that are mandatory for the PR. If we would accept the PR now, the files of the accepted BatchPreAnalysis PR would be overwritten.

Reply:
This came up as the target branch was set to "master" instead of "automation-robustness" → branch is set to "automation-robustness" now.